src/middlewared/middlewared/plugins/truecommand/update.py

```diff
@@ -50,7 +50,7 @@ async def tc_extend(self, config):
                 status_reason = 'Waiting for connection from Truecommand.'
 
         if config['remote_address']:
-            config['remote_address'] = config['remote_address'].split('/', 1)[0]
+            config['remote_address'] = f'http://{config["remote_address"].split("/", 1)[0]}/'
```

Comment:
I was thinking we could do both... IP and url, so UI doesnt have to parse either